webapi/src/main/scala/org/knora/webapi/routing/admin/ProjectsRouteADM.scala

```diff
@@ -238,17 +239,34 @@ class ProjectsRouteADM(routeData: KnoraRouteData)
         entity(as[ChangeProjectApiRequestADM]) { apiRequest => requestContext =>
           val checkedProjectIri =
             stringFormatter.validateAndEscapeProjectIri(value, throw BadRequestException(s"Invalid project IRI $value"))
+          val projectIri: ProjectIri =
+            ProjectIri.make(checkedProjectIri).getOrElse(throw BadRequestException(s"Invalid Project IRI"))
```

Comment:
```suggestion
            ProjectIri.make(checkedProjectIri).getOrElse(throw BadRequestException(s"Invalid Project IRI."))
```

Reply:
"Invalid Project IRI" is not a sentence, grammatically. So, I think the dot can be omitted. But I checked other error messages and there doesn't seem to be a consistenccy. So, I just leave it as it is.